e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/change-your-answers-change-broker.spec.js

```diff
@@ -44,8 +45,7 @@ const getFieldVariables = (fieldId, referenceNumber) => ({
   changeLink: summaryList.field(fieldId).changeLink,
 });
 
-// TODO: EMS-2793 - re-enable
-context.skip('Insurance - Check your answers - Policy - Broker - Summary list', () => {
+context('Insurance - Check your answers - Policy - Broker - Summary list', () => {
```

Comment:
should this be Change your answers - Policy - Broker - As an exporter, I want to change my answers to the broker section?